drivers/clock_control/clock_control_nrf2_hfxo.c

```diff
@@ -48,6 +76,40 @@ static void hfxo_start_up_timer_handler(struct k_timer *timer)
 	}
 }
 
+static void start_hfxo(struct dev_data_hfxo *dev_data)
+{
+	nrf_lrcconf_event_clear(NRF_LRCCONF010, NRF_LRCCONF_EVENT_HFXOSTARTED);
+	soc_lrcconf_poweron_request(&dev_data->hfxo_node, NRF_LRCCONF_POWER_MAIN);
+	nrf_lrcconf_task_trigger(NRF_LRCCONF010, NRF_LRCCONF_TASK_REQHFXO);
+}
+
+static void request_hfxo(struct dev_data_hfxo *dev_data)
+{
+	if (IS_ENABLED(CONFIG_ZERO_LATENCY_IRQS)) {
```

Comment:
this needs to be ifdef as `request_count` and `full_irq_lock`  do not exist when ZLI is not enabled.

Reply:
Hm... interesting it didn't fail when I build a sample where ZLIs were disabled.